### PR TITLE
5.6.5 Release Notes

### PR DESCRIPTION
**Pull request by andrewvc**
From git log v5.6.4...377229e
and
git diff v5.6.4 377229e

### CODE REVIEW

**Comment by colinsurprenant** [comment]
LGTM

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,12 +3,18 @@
...
 * <<logstash-5-6-4,Logstash 5.6.4>>
 * <<logstash-5-6-3,Logstash 5.6.3>>
 * <<logstash-5-6-2,Logstash 5.6.2>>
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-5]]
+=== Logstash 5.6.5 Release Notes
+* Fix bug where persistent queue could not be opened when in some states
+* Ensure logstash terminates on second sigint, there were some situations where executing two sigints would not kill Logstash
```
Comment (dedemorton) [inline docs/static/releasenotes.asciidoc]: Change to: Ensure that Logstash terminates on second SIGINT. Prior to this release, there were some situations where executing two SIGNINTs would not kill Logstash. (you might want to format SIGINT with back ticks...I can't remember whether we do elsewhere).

**Comment by dedemorton** [comment]
One other thing I didn't noticed before. We typically use past tense in the release notes: fixed, improved, etc.

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/static/releasenotes.asciidoc]: Improved shutdown process by ensuring that.... (related to my comment below about making these items past tense)

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -3,12 +3,18 @@
...
 * <<logstash-5-6-4,Logstash 5.6.4>>
 * <<logstash-5-6-3,Logstash 5.6.3>>
 * <<logstash-5-6-2,Logstash 5.6.2>>
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-5]]
+=== Logstash 5.6.5 Release Notes
+* Fixed a bug where persistent queue could not be opened when in some states
+* Improved shutdown process by ensuring that Logstash terminates on second SIGINT. Prior to this release, there were some situations where executing two SIGNINTs would not kill Logstash. (you might want to format SIGINT with back ticks...I can't remember whether we do elsewhere).
```
Comment (dedemorton) [inline docs/static/releasenotes.asciidoc]: Looks like you had some copy/paste editing weirdness. The second sentence should say: "Prior to this release, there were some situations where executing two SIGNINTs would not kill Logstash."

Make sure you remove the parenthetical statement.

**Comment by dedemorton** [comment]
LGTM

**Pull request closed without merging**